\hypertarget{classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1ResponseError}{}\doxysection{pip.\+\_\+vendor.\+urllib3.\+exceptions.\+Response\+Error Class Reference}
\label{classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1ResponseError}\index{pip.\_vendor.urllib3.exceptions.ResponseError@{pip.\_vendor.urllib3.exceptions.ResponseError}}


\doxysubsection{Detailed Description}
\begin{DoxyVerb}Used as a container for an error reason supplied in a MaxRetryError.\end{DoxyVerb}
 Inheritance diagram for pip.\+\_\+vendor.\+urllib3.\+exceptions.\+Response\+Error\+:\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[height=3.000000cm]{classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1ResponseError}
\end{center}
\end{figure}
\doxysubsection*{Static Public Attributes}
\begin{DoxyCompactItemize}
\item 
\mbox{\Hypertarget{classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1ResponseError_a9768fccf5aa140ef4c4b78e9f56b0b92}\label{classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1ResponseError_a9768fccf5aa140ef4c4b78e9f56b0b92}} 
string {\bfseries GENERIC\+\_\+\+ERROR} = \char`\"{}too many error responses\char`\"{}
\item 
\mbox{\Hypertarget{classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1ResponseError_ae8d3b4d72d0e95f47c6a1bee8bacb48a}\label{classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1ResponseError_ae8d3b4d72d0e95f47c6a1bee8bacb48a}} 
string {\bfseries SPECIFIC\+\_\+\+ERROR} = \char`\"{}too many \{status\+\_\+code\} error responses\char`\"{}
\end{DoxyCompactItemize}


The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
venv/\+Lib/site-\/packages/pip/\+\_\+vendor/urllib3/exceptions.\+py\end{DoxyCompactItemize}
